datetime/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM komand/python-pypy3-plugin:2
+FROM komand/python-3-37-plugin
```

Comment:
just want to confirm that you tried this with slim but it did not work.

Reply:
Here problem is with this line:
`RUN apt-get update && apt-get install -y gcc && rm -rf /var/lib/apt/lists/*`
I can change to slim but then I should change also this. 
Python use `maya` as dependency. 

If it is not problem to use gcc from alpine I can change this.

Reply:
@mgorny-morecode Yeah, it's fine to add any dependencies on the slim image 👍

Reply:
Update to use the slim image

Reply:
I added slim version. To do it I added `RUN apk add --no-cache --virtual build-dependencies gcc linux-headers libc-dev libffi-dev build-base openssl-dev python-dev`